Assets/Scripts/Managers/ManazoneManager.cs

```
@@ -12,7 +13,17 @@ public class ManazoneManager : MonoBehaviour
     private int mWaterManaTapped = 0;
     private int mDarknessManaTapped = 0;
     private int mFireManaTapped = 0;
-
+    
+    public bool CanPlayMana(Card _card)
+    {
```

Fail fast (consistency)

---

Resolved in branch Tony_Tap_Functionality